extensions/amp-animation/0.1/web-animation-service.js

```diff
@@ -33,14 +33,15 @@ export class WebAnimationService {
   }
 
   /**
+   * @param {!Document|!ShadowRoot|!Element} root
    * @return {!Builder}
    */
-  createBuilder() {
+  createBuilder(root) {
```

Comment:
IMHO this should definitely be `options = {root, ...}`. Once we start opening this app, there's a lot of customization.

Reply:
Sure. Removing scoping from this PR to isolate this concern.